proposals/csharp-12.0/ref-readonly-parameters.md

```diff
@@ -95,10 +99,25 @@ By-value overloads will be preferred over `ref readonly` overloads in case there
 #### Method conversions
 [method-conversions]: #method-conversions
 
-Similarly, for the purpose of anonymous function [[§10.7](https://github.com/dotnet/csharpstandard/blob/47912d4fdae2bb8c3750e6485bdc6509560ec6bf/standard/conversions.md#107-anonymous-function-conversions)] and method group [[§10.8](https://github.com/dotnet/csharpstandard/blob/47912d4fdae2bb8c3750e6485bdc6509560ec6bf/standard/conversions.md#108-method-group-conversions)] conversions, these modifiers are considered compatible (but the conversion results in a warning):
+Similarly, for the purpose of anonymous function [[§10.7](https://github.com/dotnet/csharpstandard/blob/47912d4fdae2bb8c3750e6485bdc6509560ec6bf/standard/conversions.md#107-anonymous-function-conversions)] and method group [[§10.8](https://github.com/dotnet/csharpstandard/blob/47912d4fdae2bb8c3750e6485bdc6509560ec6bf/standard/conversions.md#108-method-group-conversions)] conversions, these modifiers are considered compatible
+(but any allowed conversion between different modifiers results in a warning):
+
+- `ref readonly` can be interchanged with `in` and converted to `ref` modifier (but `ref` cannot be converted to `ref readonly`),
+- `in` can be converted to `ref` modifier, gated on LangVersion (but `ref` cannot be converted to `in`).
```

Comment:
Consider rewording as suggested for the previous bullet point. It actually feels like it would be better to have three bullet points. 
- for `ref readonly` parameter of the target method
- for `in` parameter of the target method
- for `ref` parameter of the target method